usecases.md

```diff
@@ -3,3 +3,418 @@ layout: base
+4. Forecast issue frequency
+5. Value type (e.g. mean, max, 95th percentile, instantaneous)
+
+Each of the examples below is a valid forecast:
+
+- 5 minute lead time, 5 minute interval, 1 interval per submission, 5 minute frequency
+- 1 hour lead time, 1 hour interval, 1 interval per submission, 1 hour frequency
+- 75 minute lead time, 1 hour interval, 1 interval per submission, 1 hour frequency
+- 8 hour lead time, 1 hour interval, 24 intervals per submission, 1 day frequency (hour average forecasts for next day as of 16:00 previous day)
+
+A collection of forecasts is not required to share all parameters. For example, forecasts could be compared across different lead times or interval durations.
+
+## Framework User and Framework Administrator
+
+A _framework user_ is an individual or organization that uses the capabilities of Solar Forecast Arbiter to evaluate and analyze forecasts. A framework user can be a _forecast provider_ (one who creates a forecast), a _forecast user_ (one using a forecast for a defined purpose), researchers and others.
+
+The _framework administrator_ operates Solar Forecast Arbiter, ensures data security and anonymity, establishes standards for forecast evaluation, and provides benchmark forecasts.
+
+
+
+# Use cases
+
+A use case describes a sequence of actions taken by a framework user to achieve a goal. Use cases are grouped into two categories: Evaluate forecasts, and Analyze forecasts. From each use case a list of framework requirements is developed. A use case may adopt, expand or modify the requirements of another use case.
+
+## 1. Evaluate forecasts
+
+### 1.A. Compare a forecast to measurements
+
+**Use case narrative** : A framework user uploads a forecast or selects a benchmark forecast, uploads corresponding data or selects reference data, specifies filters, specifies temporal averaging, and selects metrics. The framework calculates metrics, provides a visual display of forecast performance and an evaluation report for download.
+
+**Requirements** :
+
+- User can upload data, select from previously uploaded data, or select a location/period with reference data (4.E).
+  - If uploading data, user must define relevant metadata including location, data type, and units.
+  - Data type can be irradiance, power, or net-load.
+- User can upload a corresponding forecast or select corresponding benchmark forecast. The forecast value type must be deterministic and single-valued.
+- Framework applies data quality checks to uploaded data and forecasts (4.G).
+- User can specify filters to exclude forecast or data points (a subset of (time, value) pairs), by time or by value. (2.A).
+- User can select among forecast performance metrics (3.A).
+- Framework provides metric values and visuals of forecast performance (3.A).
+- Framework provides a report for download.
+- Framework protects forecasts and uploaded data as user-owned: user can specify other users who can see/use the uploaded forecasts and data.
```

Comment:
Aidan: add: , or summary statistics related to results. (Could this be an option? So they can’t see data but can see the results?)

Reply:
Good point.  We should add "Framework permits user to share forecast metrics and/or summary statistics for forecasts and data with other users." as a new requirement.

Reply:
I had modified the original line to read: "Framework protects forecasts and uploaded data as user-owned: user can specify other users who can see/use the uploaded forecasts, data, and summary error statistics." 

@alorenzo175 what makes sense from an implementation perspective?

Reply:
That is mostly OK. I might say "specify other users and/or groups"